recipes/openscenegraph-osgQt/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "openscenegraph-osgQt" %}
+{% set version = "3.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openscenegraph/osgQt/archive/{{ version }}.tar.gz
+  sha256: dcc4436590639168e6470fe9c5343c82bca738d3296ebee014f40f2dc029afa1
+  patches:
+    - patches/osx.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_compatible(name|lower, max_pin='x.x') }}
```

[This should be pin_subpackage](https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements):
```suggestion
    - {{ pin_subpackage(name|lower, max_pin='x.x') }}
```

---

Oh yes, sorry, forgot to change it back !